src/main/java/org/sagebionetworks/bridge/services/AccountWorkflowService.java

```diff
@@ -326,7 +326,9 @@ public void notifyAccountExists(Study study, AccountId accountId) {
         checkNotNull(accountId);
 
         Account account = accountService.getAccount(accountId);
-        
+        if (account == null) {
+            throw new EntityNotFoundException(Account.class);
+        }
```

Comment:
This can only happen if the user tries to sign up with an identifier that already exists, and then while we're trying to notify the existing account, the account becomes deleted. In this case, it would be weird for sign-up to return a 404. I think a 500 Internal Server Error would be appropriate here, or possibly a 409 Concurrent Modification Error.

Reply:
At first I was going to with this to a 409 response which makes sense to me, but that would return to the caller who might then infer that the account has existed (which we're trying to avoid)... so I think the best response here may be to return and send nothing.